opm/autodiff/BlackoilWellModel_impl.hpp

```diff
@@ -244,13 +251,25 @@ namespace Opm {
         // Compute reservoir volumes for RESV controls.
         rateConverter_.reset(new RateConverterType (phase_usage_,
                                                     std::vector<int>(number_of_cells_, 0)));
-        computeRESV(timeStepIdx);
+
+        int exception_thrown = 0;
+        try {
+            computeRESV(timeStepIdx, local_deferredLogger);
+        } catch (...) {
```

Comment:
I do not like this construct. Why the ellipsis? Can't we be more specific about what exceptions we are awaiting?

Reply:
changed